src/components/pushClient/Supportus.jsx

```diff
@@ -0,0 +1,49 @@
+import styles from './supportus.styl'
+
+import { translate } from 'cozy-ui/react/I18n'
+
+import React from 'react'
+
+import { withClient } from 'cozy-client'
+import withInstance from 'drive/web/modules/upload/withInstance'
+import withDiskUsage from './withDiskUsage'
+
+import { buildPremiumLink } from 'drive/web/modules/upload/QuotaAlert'
+import { isMobileApp } from 'cozy-device-helper'
+import get from 'lodash/get'
+
+const Supportus = ({ t, client }) => {
+  if (isMobileApp()) return null
+
+  const instanceInfo = withInstance(client)
+
+  const uuid = get(instanceInfo, 'instance.data.attributes.uuid')
+  const managerUrl = get(instanceInfo, 'context.data.attributes.manager_url')
+  const enable_premium_links = get(
+    instanceInfo,
+    'context.data.attributes.enable_premium_links'
```

Comment:
Is this set to false when the user has a premium subscription?

Reply:
added comment in https://github.com/cozy/cozy-drive/pull/1891/commits/b442343f4a21804f787cf5d64b84af6734b44a28